src/khoj/utils/cli.py

```diff
@@ -42,7 +45,9 @@ def cli(args=None):
         help="Run Khoj in anonymous mode. This does not require any login for connecting users.",
     )
 
-    args = parser.parse_args(args)
+    args, remaining_args = parser.parse_known_args(args)
+
+    logger.debug(f"Ignoring unknown args: {remaining_args}")
```

Comment:
Nit: Log message update for more context
```suggestion
    logger.debug(f"Ignoring unknown commandline args: {remaining_args}")
```